vignettes/rcmd_issues.Rmd

```diff
@@ -67,22 +67,22 @@ This will ensure that the exact package versions we use in the workflow are inst
 ## Package Dependencies
 
     > checking package dependencies ... ERROR
-      Namespace dependency not required: ‘pkg’
+      Namespace dependency not required: ‘pkg'
```

Comment:
I assume this change was a mistake. Same for line 77 and 85 below.

Reply:
fixed up!!